google-cloud-logging/src/main/java/com/google/cloud/logging/MonitoredResourceUtil.java

```diff
@@ -40,10 +40,13 @@
     ContainerName("container_name"),
     InstanceId("instance_id"),
     InstanceName("instance_name"),
+    Location("location"),
```

Comment:
Is there a reason to use "location" instead of "region"?

Reply:
I looked at log entries that get ingested from stdout, and they all have the `location` attribute. So, I tried to match that.